sdk/ai/azure-ai-projects-dp1/samples/deployments/sample_deployments_async.py

```diff
@@ -0,0 +1,56 @@
+
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    Given an AIProjectClient, this sample demonstrates how to use the asynchronous
+    `.deployments` methods to enumerate AI models deployed to your AI Foundry Project.
+
+USAGE:
+    python sample_deployments_async.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity aiohttp
+
+    Set these environment variables with your own values:
+    1) PROJECT_ENDPOINT - Required. The Azure AI Project endpoint, as found in the overview page of your
+       Azure AI Foundry project.
+"""
+
+import asyncio
+import os
+from azure.identity.aio import DefaultAzureCredential
+from azure.ai.projects.dp1.aio import AIProjectClient
+
+async def sample_deployments_async() -> None:
+
+    endpoint = os.environ["PROJECT_ENDPOINT"]
+
+    project_client = AIProjectClient(
+        endpoint=endpoint,
+        credential=DefaultAzureCredential(),
+    )
+
+    print("Get a single deployment by its name:")
+    deployment = await project_client.deployments.get("gpt-4o-mini")
```

Comment:
Do we want this in ENV variable?

Reply:
I made another commit to read the deployment name and model publisher name from env variable. Thanks!